doc/articles/platform-specific-xaml.md

```diff
@@ -0,0 +1,70 @@
+# Platform-specific XAML markup in Uno
```

Comment:
You may want to remove the section that deals with that topic here: https://github.com/unoplatform/uno/blob/master/doc/articles/using-uno-ui.md#supporting-multiple-platforms-in-xaml-files

Reply:
I did :) (Just left a redirect to not completely break existing bookmarks)